### DISCUSSION TITLE
Permit Markdown in profile biographies.

### DISCUSSION THREAD

**Discussion by beedell-roke** [community/community#119681] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

As https://github.com/github-tools/github/issues/344#issue-158414627:~:text=the%20new%20bio%20system%20doesn't%20support%20markdown requests, permit Markdown in [the profile biography](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile#adding-a-bio-to-your-profile). [^1]

[^1]: [`github.com/github/docs/blob/a39f386d0b3958fafc14d4ba2aa53912e1a20a74/content/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile.md#adding-a-bio-to-your-profile`](https://github.com/github/docs/blob/a39f386d0b3958fafc14d4ba2aa53912e1a20a74/content/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile.md#adding-a-bio-to-your-profile:~:text=on%20your%20profile.-,Adding%20a%20bio%20to%20your%20profile,-Add%20a%20bio)

**Comment by beedell-roke** [comment]
> [`#discussion-6509560`](https://github.com/orgs/community/discussions/119681#discussion-6509560)

@DanielRuf, is there any chance that you could post a URI to this at https://github.com/github-tools/github/issues/344#issuecomment-625395171, so that those subscribed can see that there is now an official request? I ask because https://github.com/github-tools/github/issues/344#issue-158414627 still appears in the first page of [`google.com/search?q=bio+markdown+support+github`](https://www.google.com/search?q=bio+markdown+support+github).

**Reply by beedell-roke** [reply]
> [`#discussioncomment-10512934`](https://github.com/orgs/community/discussions/119681#discussioncomment-10512934)

I've created https://github.com/dear-github/dear-github/issues/437#issue-2752898882 in an attempt to link them, but locked issues appear to not display linkages, subsequent to being locked. If [`gitlab.com/gitlab-org/gitlab/-/merge_requests/35604#note_3084468047`](https://gitlab.com/gitlab-org/gitlab/-/merge_requests/35604#note_3084468047) gains any traction, GitHub might care.

**Comment by jacopotediosi** [comment]
I'm against this one because I don't want to get distracted by people biographies and their weird styles.

**Reply by beedell-roke** [reply]
@jacopotediosi, that's a fault of GitHub's CSS, then. More semantic markup is more accessible.
